website/docs/guides/language-specific-tutorials/django/implementing-feature-flags-django.md

```diff
@@ -43,11 +43,11 @@ Password: unleash4all
 
 Click the 'New feature flag' button to create a new feature flag.
 
-![Create a new feature flag](../ruby/new-ff.png)
+![Create a new feature flag](/img/ruby-guide-new-ff.png)
```

Comment:
There's been some image reuse across these language-specific guides. For now, I moved them into the /img repo and prefixed them with the framework name, but it's still not ideal. Since they are also now mostly outdated screenshots, it makes sense to update and consolidate them in a separate iteration. I'll apply better naming then